vogue/commands/load/analysis.py

```diff
@@ -1,16 +1,52 @@
+import logging
+
 import click
 
+import yaml
+import json
+
+from vogue.tools.cli_utils import json_read, yaml_read, check_file
+
+LOG_LEVELS = ['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL']
+LOG = logging.getLogger(__name__)
+
+def validate_conf(analysis_conf):
+    """
+    Takes input analysis_conf dictionary and validates entries.
+    """
+    return True
+
 @click.command("analysis", short_help = "Read files from analysis workflows")
 @click.option(
     '-s',
     '--sample-id',
     required=True,
     help='Input sample id'
 )
+@click.option(
+    '-a',
+    '--analysis-config',
+    type=click.Path(),
+    required=True,
+    help='Input config file. Accepted format: JSON, YAML')
 @click.pass_context
 
-def analysis(context, sample_id):
+def analysis(context, sample_id, analysis_config):
     """
     Read and load analysis results. These are either QC or analysis output files.
     """
-    pass
+    LOG.info("Reading and validating config file.")
+    check_file(analysis_config)
```

Comment:
```suggestion
    try:
        check_file(analysis_config)
    except FileNotFoundError as err:
        context.abort()
```
Sorry if I'm bugging you here but this will not exit gratefully, right?

Reply:
Good point! Thanks for pointing it out.